docs/development.md

```diff
@@ -157,6 +157,8 @@ git clone git@github.com:ansible/eda-server.git
 
 ### Install dependencies
 
+**NOTE**: Since we added experimental [LDAP authentication](https://github.com/ansible/eda-server/pull/557), we have new dependencies such as `xmlsec` for which you may need to install the required [native libraries first](https://pypi.org/project/xmlsec/).
```

Comment:
Let's make the things easier just indicating the packages that we need, don't? 
fedora:
openldap-devel libxml2-devel xmlsec1-devel xmlsec1-openssl-devel libtool-ltdl-devel

Reply:
updated.